weblate/checks/consistency.py

```diff
@@ -98,6 +98,18 @@ class TranslatedCheck(TargetCheck):
     description = _("This string has been translated in the past")
     ignore_untranslated = False
 
+    def get_description(self, check=None):
+        unit = check.unit
+        target = self.check_target_unit(unit.source, unit.target, unit)
+        self.description = (
```

Comment:
Why updating description attribute? I see no reason for that (and it will break things).

Reply:
Sorry, missed it before. Changed now.